xml/operations-maintenance-controller-pit_database_recovery.xml

```diff
@@ -8,132 +8,146 @@
-ardana-cp1-c1-m1</screen>
-   </listitem>
-   <listitem>
+<screen>cat /etc/hosts | grep ardana002-cp1-c1-m1
+10.84.43.82      ardana002-cp1-c1-m1-extapi ardana002-cp1-c1-m1-extapi
+192.168.24.21    ardana002-cp1-c1-m1-mgmt ardana002-cp1-c1-m1-mgmt
+10.1.2.1         ardana002-cp1-c1-m1-guest ardana002-cp1-c1-m1-guest
+10.84.65.3       ardana002-cp1-c1-m1-EXTERNAL-VM ardana002-cp1-c1-m1-external-vm
+    </screen>
     <para>
-     Log in to the first host member from the <literal>FND-MDB</literal> group.
+     In this example, <literal>192.168.24.21</literal> is the IP address for
+     the host.
     </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     Become root:
+     SSH into the host.
     </para>
-<screen>sudo su</screen>
-   </listitem>
-   <listitem>
+<screen>ssh ardana@192.168.24.21</screen>
+   </step>
+   <step>
     <para>
-     Source the backup environment file
+     Source the backup file.
     </para>
-<screen>source /home/stack/backup.osrc</screen>
-   </listitem>
-   <listitem>
+<screen>source /var/lib/ardana/backup.osrc</screen>
+   </step>
+   <step>
+    <para>
+     Find the client id.
+    </para>
+<screen>
+<?dbsuse-fo font-size="0.65em"?>
+freezer client-list
++-----------------------------+----------------------------------+-----------------------------+-------------+
+| Client ID                   | uuid                             | hostname                    | description |
++-----------------------------+----------------------------------+-----------------------------+-------------+
+| ardana002-cp1-comp0001-mgmt | f4d9cfe0725145fb91aaf95c80831dd6 | ardana002-cp1-comp0001-mgmt |             |
+| ardana002-cp1-comp0002-mgmt | 55c93eb7d609467a8287f175a2275219 | ardana002-cp1-comp0002-mgmt |             |
+| ardana002-cp1-c0-m1-mgmt    | 50d26318e81a408e97d1b6639b9404b2 | ardana002-cp1-c0-m1-mgmt    |             |
+| ardana002-cp1-c1-m1-mgmt    | 78fe921473914bf6a802ad360c09d35b | ardana002-cp1-c1-m1-mgmt    |             |
+| ardana002-cp1-c1-m2-mgmt    | b2e9a4305c4b4272acf044e3f89d327f | ardana002-cp1-c1-m2-mgmt    |             |
+| ardana002-cp1-c1-m3-mgmt    | a3ceb80b8212425687dd11a92c8bc48e | ardana002-cp1-c1-m3-mgmt    |             |
++-----------------------------+----------------------------------+-----------------------------+-------------+</screen>
+    <para>
+     In this example, the client id of host
+     <literal>ardana002-cp1-c1-m1-mgmt</literal> is
+     <literal>ardana002-cp1-c1-m1-mgmt</literal>.
+    </para>
+   </step>
+   <step>
     <para>
      List the jobs
     </para>
-<screen>freezer-scheduler -c &lt;hostname&gt; job-list</screen>
-   </listitem>
-   <listitem>
+<screen>freezer job-list -C ardana002-cp1-c1-m1-mgmt</screen>
+   </step>
+   <step>
     <para>
-     Get the id corresponding to the job <literal>&lcm; Default: Mysql restore
+     Get the corresponding job id for <literal>Ardana Default: mysql restore
      from Swift</literal>.
     </para>
```

Comment:
This sentence does not make sense to me. Is "Ardana Default: mysql restore from Swift" a column from the output of the previous command? Then we should name it as such (Get the job ID from the column / Get the job ID from the line starting with", and this text should be part of the previous step.

Reply:
the statement is based on the output 
+----------------------------------+---------------------------------------------+-----------+---------+-----------+-------+------------+
| Job ID                           | Description                                 | # Actions | Result  | Status    | Event | Session ID |
+----------------------------------+---------------------------------------------+-----------+---------+-----------+-------+------------+
| 35c7d3341b134b95b44dbd4edf647928 | Ardana Default: mysql restore from Swift    |         1 |         | stop      |       |            |
| ad79377a6c164132be14ee4fb2f2d024 | Ardana Default: deployer restore from Swift |         7 |         | stop      |       |            |
| 10ce154e0cd64ace82dc3d2df1ec19d5 | Ardana Default: swift restore from Swift    |         1 |         | stop      |       |            |
| 9ce509c3eb064b3a93010cb1dfe7a860 | Ardana Default: audit restore from Swift    |         1 |         | stop      |       |            |
| f95b23257e534e93bdff48ac44462212 | Ardana Default: deployer backup to Swift    |         7 | success | scheduled |       |            |
| a6accca3eaf4474e914483dddf40621f | Ardana Default: mysql backup to Swift       |         1 | success | scheduled |       |            |
| 42712d3019ed4f0581c650c6ffe07a87 | Ardana Default: swift backup to Swift       |         1 | success | scheduled |       |            |
| ae8344d7b1a440cdbcfb46439e6d31b3 | Ardana Default: audit backup to Swift       |         1 | success | scheduled |       |            |
+----------------------------------+---------------------------------------------+-----------+---------+-----------+-------+------------+